task1.py

```diff
@@ -0,0 +1,8 @@
+numbers = [2, -93, -2, 8, None, -44, -1, -85, -14, 90, -22, -90, -100, -8, 38, -92, -45, 67, 53, 25]
+numbers[4]=1
+sum_ = (sum(numbers) - 1)
+len_ = len(numbers)
+average = round(sum_/len_ ,2)
```

Comment:
```suggestion
average = sum_ // len_ 
```
Используя оператор целочисленного остатка от деления, отсутствует необходимость в округлении. Также при округлении есть вероятность получить значение, округленное в большую сторону, что приведет к ошибке. 

Также не забывайте об оформлении кода - расстановке пробелов до и после математических операторов